### PR TITLE
[MODULAR] Fixes Subtle and Subtler Swaps

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
Now subtle and subtler use a local variable to store the message that's being sent, instead of a global variable between all of them.

In layman's terms, the emote will now properly remember your emote and will not let it get overwritten by someone else's.

~~Do note that this needs to be test-merged first, I don't see why it wouldn't work, but my test server broke and so I wasn't able to properly test this out.~~
It has been tested. It is ready.

## Why It's Good For The Game
No more getting someone else's paragraph describing how they're holding hands instead of your own.

## Changelog
:cl:GoldenAlpharex
fix: Subtle and subtler should no longer be confused and display someone else's message. Your emotes shall be safe again.
/:cl:

### CODE REVIEW

**Comment by xXPawnStarrXx** [comment]
Now I can lewd privately without someone getting access to my messages about ██████████████████████████████████████████████████ or even worse ████████████████████████████.

**Comment by Arctaisia** [comment]
Gone are the days of someone getting my paragraphs of v██████████████████.

**Review by Funce — APPROVED** [review]
I'm surprised. I don't think emotes were ever meant to have mutable message strings...

Good find on this.

**Comment by KasparoVy** [comment]
Awesome job figuring this out, wow.

**Comment by GoldenAlpharex** [comment]
> Awesome job figuring this out, wow.

I know, I know ;)
